src/comments/entities/comment.entity.ts

```diff
@@ -0,0 +1,39 @@
+import { BlogEntity } from '../../blogs/model/blog.entity';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  Tree,
+  UpdateDateColumn,
+} from 'typeorm';
+
+@Entity('comments')
+export class CommentEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ type: 'varchar', length: 250 })
+  title: string;
+
+  @Column()
+  is_published: boolean;
+
+  @CreateDateColumn()
+  created_at: Date;
+
+  @UpdateDateColumn()
+  updated_at: Date;
+
+  @Column({ type: 'varchar' })
+  content: string;
+
+  @ManyToOne(() => BlogEntity, blog => blog.id)
+  @JoinColumn({ name: 'blog_id' })
+  blog_id: BlogEntity;
+
+  @Column({ nullable: true })
+  parent_id: string;
+}
```

Comment:
Are you going to use  the`Tree` that is imported above?, this you should consider wether you need this column or not according to the method you will pattern you will use to implement the tree structure.